components/home/index.js

```diff
@@ -29,11 +31,16 @@ export default class Home extends Component {
     });
   }
 
+  changeVisibilityTo = (boolie) => {
+    this.setState({ isVisible: boolie });
+  }
+
   render() {
     return (
       <View style={styles.container}>
         <TheMap updatedRegion={this.state.region} />
-        <SearchBar callBack={this.updateRegion} />
+        <SearchBar callBack={this.updateRegion} changeVisibilityTo={this.changeVisibilityTo} />
+        {this.state.isVisible && <SwitchCampuses callBack={this.updateRegion} />}
```

Comment:
Can this be inside the searchBar Component?  it seems that that is where it belongs